specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/examples/ServiceValidateAddress.json

```diff
@@ -1,45 +1,29 @@
 {
   "parameters": {
-    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "subscriptionId": "3c66da21-607e-49b4-8bdf-f25fbb8f705f",
```

Comment:
Are we planning to use new subscription instead of BVT subscription?

Reply:
I have modified the subscription to the old one.